doc/avifenc.1.md

```diff
@@ -189,7 +189,7 @@ The following options can optionally have a **:u** (or **:update**) suffix like
 **\--autotiling**
 :   Set \--tilerowslog2 and \--tilecolslog2 automatically.
     If specified, switch to automatic tiling.
-    avifenc starts in manual tiling mode.
+    avifenc starts in automatic tiling mode.
```

Comment:
The date of the man page could be updated too.

Reply:
Done.